artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/management/ActiveMQServerControl.java

```diff
@@ -1988,11 +1988,27 @@ void resetUser(@Parameter(name = "username", desc = "Name of the user") String u
    @Operation(desc = "forces the broker to reload its configuration file", impact = MBeanOperationInfo.ACTION)
    void reloadConfigurationFile() throws Exception;
 
+   /**
```

Comment:
I'm adding javadoc here, since I am adding a hyperlink from the doc towards the javadoc.